app/code/Magento/Catalog/Model/Product/Option.php

```diff
@@ -135,12 +148,16 @@ public function __construct(
         \Magento\Framework\Model\ResourceModel\AbstractResource $resource = null,
         \Magento\Framework\Data\Collection\AbstractDb $resourceCollection = null,
         array $data = [],
+        array $groupsPool = [],
```

Comment:
All new constructor arguments should be added at the end. Please fix that

Reply:
Affirmative